tests/jit.rs

```diff
@@ -118,7 +118,7 @@ fn test_code_length_estimate() {
                     opcode = 0x85;
                     (0x00, Some(0x91020CDD))
                 }
-                0xF5 => {
+                0xF5 if !sbpf_version.static_syscalls() => {
```

Comment:
This test bypasses verification, which would otehrwise have prevented this case (invalid function calls in SBPFv3).